LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -483,7 +512,13 @@ $(function() {
 	dialog_simple = new dialogController('simplexhrDialog', 'simplexhr', 'simpleloadIndicator', 'simplebutton_save', 'simplebutton_cancelchange');
 
 	$('#simpleloadIndicator').css({width: $(window).width() * .78, height: $(window).height() * .78});
+
+    dialog_simple.setCancelHandler(function(){
+        $('#simplexhr').css({width: $(window).width() * .8, height: $(window).height() * .8});
+        dialog_simple.setTitle("");
+    });
 	$('#simplexhr').css({width: $(window).width() * .8, height: $(window).height() * .8});
+    $('#simplexhrDialog').dialog({minWidth: ($(window).width() * .78) + 30});
```

Comment:
indentation?

Reply:
This has dev merged into it. So there are some changes not related to routing. This is the indentation present in dev.